integration/testdata/ca_cert_apps/no_package_manager/app.py

```diff
@@ -1,6 +1,21 @@
 import http.server, ssl
+from http import HTTPStatus
+
+class Handler(http.server.SimpleHTTPRequestHandler):
+    def do_GET(self):
+        self.send_response(HTTPStatus.OK)
+        self.end_headers()
+        self.wfile.write(b'Powered by Paketo Buildpacks')
 
 server_address = ('0.0.0.0', 8080)
-httpd = http.server.HTTPServer(server_address, http.server.SimpleHTTPRequestHandler)
-httpd.socket = ssl.wrap_socket(httpd.socket, server_side=True, certfile='cert.pem', keyfile='key.pem', ssl_version=ssl.PROTOCOL_TLS)
+httpd = http.server.HTTPServer(server_address, Handler)
+httpd.socket = ssl.wrap_socket(
+    httpd.socket,
+    ca_certs='/bindings/ca-certificates/ca.pem',
```

Comment:
You shouldn't be referring to the ca certificate directly that's volume-mounted.
The integration test is making use that the python buildpack (via its constituent ca-cert cnb) loaded the certificate present at `/bindings/ca-certificates/ca.pem` to the system truststore. So the test app should make use of the system trust store via python's API for it